video_common.c:436:5: warning: "/*" within comment

Compiling `4.0.1+git37b3595`
```
video_common.c: In function 'y10torgb24':
video_common.c:436:5: warning: "/*" within comment [-Wcomment]
     /* url: http://salottisipuli.retiisi.org.uk/cgi-bin/gitweb.cgi?p=~sailus/raw2rgbpnm.git;a=summary */
 ^
```

--- a/motion.c
+++ b/motion.c
@@ -591,7 +591,7 @@ static void process_image_ring(struct context *cnt, unsigned int max_images)
                     if (cnt_list[0]->log_level >= DBG) {
                         int frames = cnt->movie_fps - (cnt->movie_last_shot + 1);
                         if (frames > 0) {
-                            char tmp[15];
+                            char tmp[25];
                             MOTION_LOG(DBG, TYPE_ALL, NO_ERRNO, "%s: Added %d fillerframes into movie",
                                        frames);
                             sprintf(tmp, "Fillerframes %d", frames);
@@ -1698,7 +1698,7 @@ static void *motion_loop(void *arg)
 
             /* Add changed pixels in upper right corner of the pictures */
             if (cnt->conf.text_changes) {
-                char tmp[15];
+                char tmp[25];
 
                 if (!cnt->pause)
                     sprintf(tmp, "%d", cnt->current_image->diffs);

--- a/motion_guide.html
+++ b/motion_guide.html
@@ -196,6 +196,19 @@ Motion is part of the Ubuntu and Debian repositories and can be installed with t
 <p></p>
 Open up a terminal window and type: <code><strong>sudo apt-get install motion</strong></code>
 <p></p>
+Motion can also be installed from the release deb files.  First, get the link to the deb file
+for the distribution and platform from the <a href="https://github.com/Motion-Project/motion/releases">Releases</a>
+page.
+<p></p>
+Open up a terminal window and type:
+<code><strong>wget https://github.com/Motion-Project/motion/releases/{deb package name} </strong></code>
+Next, there are multiple options to install a deb package, the below is one option
+and uses the gdebi tool.
+<p></p>
+<code><strong>sudo apt-get install gdebi-core</strong></code>
+<p></p>
+<code><strong>sudo gdebi {deb package name}</strong></code>
+<p></p>
 Before we start configuring Motion, we will copy the config file to our Home folder so that the master copy
 won't be affected. Open a terminal and copy the configuration file to your Home folder with following commands:
 <p></p>

--- a/video_common.c
+++ b/video_common.c
@@ -432,7 +432,7 @@ int mjpegtoyuv420p(unsigned char *map, unsigned char *cap_map, int width, int he
  */
 void y10torgb24(unsigned char *map, unsigned char *cap_map, int width, int height, int shift)
 {
-    /* Source code: raw2rgbpnm project
+    /* Source code: raw2rgbpnm project */
     /* url: http://salottisipuli.retiisi.org.uk/cgi-bin/gitweb.cgi?p=~sailus/raw2rgbpnm.git;a=summary */
 
     /* bpp - bits per pixel */
@@ -820,7 +820,7 @@ static int vid_v4lx_start(struct context *cnt)
         dev->height = height;
 #endif
 
-#if defined(HAVE_LINUX_VIDEODEV_H) && (!defined(WITHOUT_V4L))      
+#if defined(HAVE_LINUX_VIDEODEV_H) && (!defined(WITHOUT_V4L))
         if (!v4l_start(dev, width, height, input, norm, frequency, tuner_number)) {
             close(dev->fd);
             pthread_mutexattr_destroy(&dev->attr);
@@ -894,7 +894,7 @@ static int vid_v4lx_start(struct context *cnt)
  * Returns
  *     device number
  *     -1 if failed to open device.
- *     -3 image dimensions are not modulo 8 
+ *     -3 image dimensions are not modulo 8
  */
 int vid_start(struct context *cnt)
 {
@@ -1005,10 +1005,10 @@ int vid_next(struct context *cnt, unsigned char *map)
             ret = v4l2_next(cnt, dev, map, width, height);
         } else {
 #endif
-#if defined(HAVE_LINUX_VIDEODEV_H) && (!defined(WITHOUT_V4L))           
+#if defined(HAVE_LINUX_VIDEODEV_H) && (!defined(WITHOUT_V4L))
             v4l_set_input(cnt, dev, map, width, height, conf);
             ret = v4l_next(dev, map, width, height);
-#endif            
+#endif
 #ifdef MOTION_V4L2
         }
 #endif

